src/Properties/SchemaAggregator.php

```diff
@@ -3,12 +3,12 @@
 declare(strict_types=1);
 
 /**
- * This file is part of Larastan.
+ * This file uses parts of code from following projects.
```

Comment:
I would remove this too.

Reply:
Maybe just add a note on the class phpdocs.

Reply:
@nunomaduro There is such a thing as PHPDoc file comment.

```php
/**
 * @package Larastan
 * @license GPL
 */
```

and so on https://docs.phpdoc.org/latest/references/phpdoc/index.html

Reply:
Nah, I rather keep it simple with a link to the copy/pasted code.

Reply:
Updated.